backend.native/tests/debugger/src/test/kotlin/org/jetbrains/kotlin/native/test/debugger/Matchers.kt

```diff
@@ -69,13 +69,54 @@ fun lldbTest(@Language("kotlin") programText: String, lldbSession: String) {
     val source = tmpdir.resolve("main.kt")
     val output = tmpdir.resolve("program.kexe")
 
-    val driver = ToolDriver(DistProperties.konanc, DistProperties.lldb, DistProperties.lldbPrettyPrinters)
+    val driver = ToolDriver(DistProperties.konanc, DistProperties.lldb, DistProperties.dwarfDump, DistProperties.lldbPrettyPrinters)
     Files.write(source, programText.trimIndent().toByteArray())
     driver.compile(source, output, "-g")
     val result = driver.runLldb(output, lldbSessionSpec.commands)
     lldbSessionSpec.match(result)
 }
 
+fun dwarfDumpTest(@Language("kotlin") programText: String, flags:List<String>, test:List<DwarfTag>.()->Unit) {
```

Comment:
Here and in other places there are incosnsistancy in formatting the parameters and val|vars. They should have a space after the `:`